packages/playground/src/components/manage_caprover_worker_dialog.vue

```diff
@@ -17,10 +17,17 @@
           { title: 'PLACEHOLDER', key: 'data-table-select' },
           { title: 'Contract ID', key: 'contractId' },
           { title: 'Name', key: 'name' },
-          { title: 'Public IPv4', key: 'publicIP.ip' },
-          { title: 'CPU(vCores)', key: 'capacity.cpu' },
-          { title: 'Memory(MB)', key: 'capacity.memory' },
-          { title: 'Disk(GB)', key: 'disk' },
+          {
+            title: 'Networks',
+            key: 'networks',
+            sortable: false,
+            children: [
+              { title: 'Public IPv4', key: 'publicIP.ip', sortable: false },
+              { title: 'MyCelium IP', key: 'myceliumIP', sortable: false },
```

Comment:
```suggestion
              { title: 'Mycelium IP', key: 'myceliumIP', sortable: false },
```
so it can be consistent with all the other solutions

Reply:
@AhmedHanafy725 say something :D , here and https://github.com/threefoldtech/tfgrid-sdk-ts/pull/3590#discussion_r1857138965

Reply:
`Mycelium`